Frontend/ui-customer/src/Screens/OnBoardingFlow/EnterMobileNumberScreen/View.purs

```diff
@@ -151,25 +152,24 @@ enterMobileNumberView  state lang push =
             , width MATCH_PARENT
             , visibility  if state.props.countryCodeOptionExpended then GONE else VISIBLE
             ][PrimaryButton.view (push <<< MobileNumberButtonAction) (mobileNumberButtonConfig state)]
-            -- Until Backend Whatsapp OTP DONE
-        -- , PrestoAnim.animationSet [Anim.fadeOut state.props.countryCodeOptionExpended] $ textView $ [
-        --     height WRAP_CONTENT
-        --   , width MATCH_PARENT
-        --   , text $ getString OR
-        --   , gravity CENTER
-        --   , margin $ MarginVertical 24 24
-        --   , color Color.black500
-        --   , visibility  if state.props.countryCodeOptionExpended then GONE else VISIBLE
-        --   ] <> FontStyle.tags TypoGraphy
-        -- , PrestoAnim.animationSet
-        --   [ Anim.fadeIn $ not state.props.enterOTP
-        --   , Anim.fadeOut $ state.props.countryCodeOptionExpended || state.props.enterOTP
-        --   ] $
-        --   linearLayout
-        --     [ height WRAP_CONTENT
-        --     , width MATCH_PARENT
-        --     , visibility  if state.props.countryCodeOptionExpended then GONE else VISIBLE
-        --     ][PrimaryButton.view (push <<< WhatsAppOTPButtonAction) (whatsAppOTPButtonConfig state)]
+        , PrestoAnim.animationSet [Anim.fadeOut state.props.countryCodeOptionExpended] $ textView $ [
+            height WRAP_CONTENT
+          , width MATCH_PARENT
+          , text $ getString OR
+          , gravity CENTER
+          , margin $ MarginVertical 24 24
+          , color Color.black500
+          , visibility  if state.props.countryCodeOptionExpended || (getMerchant FunctionCall) /= YATRISATHI then GONE else VISIBLE
+          ] <> FontStyle.tags TypoGraphy
+        , PrestoAnim.animationSet
+          [ Anim.fadeIn $ not state.props.enterOTP
+          , Anim.fadeOut $ state.props.countryCodeOptionExpended || state.props.enterOTP
+          ] $
+          linearLayout
+            [ height WRAP_CONTENT
+            , width MATCH_PARENT
+            , visibility  if state.props.countryCodeOptionExpended then GONE else VISIBLE
```

Comment:
@S-PRAVINKUMAR use this condition to call the linear layout itself, instead of changing the visibility.